updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -7,18 +7,18 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 Due to fundamental Kubernetes design, all {product-title} updates between minor versions must be serialized.
-You must update from {product-title} 4.9 to 4.10 and then to 4.11. You cannot update from {product-title} 4.9 to 4.11 directly.
-However, beginning with the update from {product-title} 4.9 to 4.10 to 4.11, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-control plane hosts.
+You must update from {product-title} 4.10 to 4.11 and then to 4.12. You cannot update from {product-title} 4.10 to 4.12 directly.
+However, beginning with the update from {product-title} 4.10 to 4.11 to 4.12, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-control plane hosts.
 
 There are a number of caveats to consider when attempting an EUS-to-EUS update.
 
 * EUS-to-EUS updates are only offered after updates between all versions involved have been made available in `stable` channels.
 * If you encounter issues during or after upgrading to the odd-numbered minor version but before upgrading to the next even-numbered version, then remediation of those issues may require that non-control plane hosts complete the update to the odd-numbered version before moving forward.
 * You can do a partial update by updating the worker or custom pool nodes to accommodate the time it takes for maintenance.
 * You can complete the update process during multiple maintenance windows by pausing at intermediate steps. However, plan to complete the entire update within 60 days. This is critical to ensure that normal cluster automation processes are completed including those associated with certificate rotation.
-* You must be running at least {product-title} 4.8.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.8.z before attempting the EUS-to-EUS update.
+* You must be running at least {product-title} 4.10.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.10.z before attempting the EUS-to-EUS update.
```

Comment:
Same comment as above - check if this is still relevant for 4.12

Reply:
@petr-muller @wking what do y'all think? :)

Reply:
removed per @wking 's comment :)